fnConsensus/reactor.go

```diff
@@ -749,6 +749,13 @@ func (f *FnConsensusReactor) handleMaj23VoteSetChannel(sender p2p.Peer, msgBytes
 		return
 	}
 
+	if f.cfg.FnConcensusSignerCfg.Enabled {
+		// check if this node part of validator, if not broadcast the msg
+		if f.cfg.FnConcensusSignerCfg.Validator == false {
+			f.broadcastMsgSync(FnVoteSetChannel, nil, msgBytes)
+		}
+	}
```

Comment:
Should probably create a new function for this called `forwardMaj23VoteSet`, because you don't want to be locking the `stateMtx` or accessing the validator set.